### ISSUE TITLE
PHP manual installation on Mac OS High Sierra 

### ISSUE THREAD

**Issue by rohanpande** [bobthecow/psysh#464]
Hello Sir, 
Thank you for this amazing library. 
I was trying to in-corporate php manual into psysh so that I could get all documentation information

i.e. doc array_walk 

However as I followed the instruction specified in 
 - https://github.com/bobthecow/psysh/wiki/PHP-manual 

I am getting following view

![screen shot 2018-01-20 at 11 27 46 am](https://user-images.githubusercontent.com/3070229/35180441-94edc392-fdd6-11e7-9915-53fc7279796f.png)

I tried searching on google for this issue, but didn't came across any solution. 

Could you please help me fix this. 

Thank you

**Comment by bobthecow** [comment]
Those are color escape codes. Somehow PsySH thinks you're in a terminal emulator and shell that knows about color, but your terminal emulator and shell aren't interpreting the color codes correctly. It's probably unrelated to the manual itself, though.

If you run `ls -al --functions` do you see color or escape codes in the output?

**Comment by rohanpande** [comment]
Hello sir, 
Sorry for the delay in reply, 

When I do ls -al --functions I got the escape codes. 

- Currently I have disabled the 'colorMode' => \Psy\Configuration::COLOR_MODE_DISABLED as my Iterm2 terminal & default terminal too could not able to escape colorCodes properly. 

- After disabling colorMode - all texts became readable 

- I am using Meslo LG S DZ for powerline font 

- I have installed zsh shell 

Once again, Thank you for this wonderful library. It is helping a lot in learning core php. 

Thank you
